cloudpickle/__init__.py

```diff
@@ -1,5 +1,12 @@
 from __future__ import absolute_import
 
-from cloudpickle.cloudpickle import *
+import sys
+import pickle
+
+
+if sys.version_info[:2] >= (3, 8):
+    from cloudpickle.cloudpickle_fast import *
+else:
+    from cloudpickle.cloudpickle import *
```

Comment:
Maybe instead we should instead do:

```
from cloudpickle.cloudpickle import *
if sys.version_info[:2] >= (3, 8):
    # Use the faster C-based pickler implementation
    from cloudpickle.cloudpickle_fast import CloudPickler, dumps, dump
```

There is no need to pollute the top level namespace with the c-based helper functions.

The line

```python
from cloudpickle.cloudpickle import *
```
can be left unchanged for backward compat.